Independer.WCFDataAnnotations/ValidateDataAnnotationsBehavior.cs

```diff
@@ -82,12 +73,12 @@ public class ValidateDataAnnotationsBehavior : Attribute, IServiceBehavior {
 
       var contractOperations = serviceHostBase.Description.Endpoints.SelectMany(x => x.Contract.Operations).ToList();
 
+      var errorMessageGenerator = new ErrorMessageGenerator();
+
       foreach (var operation in operations) {
         var parameterInfo = GetParameterInfo(operation.Name, contractOperations);
 
-        operation.ParameterInspectors.Add(parameterInfo.HasAnyParameterSkipNullCheck
-          ? new ValidatingParameterInspector(_validators, new ErrorMessageGenerator(), _validationResultsLogger, parameterInfo)
-          : _defaultValidatingParameterInspector);
+        operation.ParameterInspectors.Add(new ValidatingParameterInspector(_validators, errorMessageGenerator, _validationResultsLogger, parameterInfo));

```

Comment:
did you get rid of HasAnyParameterSkipNullCheck on purpose here?

Reply:
yes thus, since this is now set on the parameterDetails.SkipNullCheck, correct?

Reply:
In the initial approach I only needed the parameterInfo if we used the AllowNull attribute, but for the parameter name addition I need to pass it always.